SU-SWS/decanter
# Use svg icons as "mask" so their colors could be changed using CSS background-color

# READY FOR REVIEW

# Summary
- Investigate best way to use SVG icons
https://github.com/SU-SWS/decanter/issues/423
- Use SVG icons as mask seems to do the trick (except IE). After discussion with Kerri and JB, we think it's ok to just hide the icon in IE.
- This is just a test so I've only done work on .su-link--external. If this way of including SVG icon is approved, we'll refactor the other components where SVG icons are used in another PR.

# Needed By (Date)
- Sooner the better since Kerri/JB is working on the new link components this week.

# Urgency
- Earlier the better :)

# Steps to Test

1. Pull this branch and compile styleguide
2. Go to Components - links, and check that the external link variant icon works fine in all browsers except IE, and that the icon is hidden in IE.

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- https://github.com/SU-SWS/decanter/issues/423
